### PR TITLE
improvement of the article "How to Return Multiple Values From a Java…

### PR DESCRIPTION
**Pull request by sk1418**
… Method"

### CODE REVIEW

File: core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsPair.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.Comparator;
+import java.util.List;
+
+class MultipleReturnValuesUsingApacheCommonsPair {
+
+    static ImmutablePair<Coordinates, Double> getMostDistantPoint(
+        List<Coordinates> coordinatesList,
+        Coordinates target) {
+        return coordinatesList.stream()
+                              .map(coordinates -> ImmutablePair.of(coordinates, coordinates.calculateDistance(target)))
```
Comment (eric-martin) [inline core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsPair.java]: When continuing a line, we use a two space indent

File: core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsTriple.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.stream.Collectors;
+
+class MultipleReturnValuesUsingApacheCommonsTriple {
+
+    static ImmutableTriple<Double, Double, Double> getMinAvgMaxTriple(
+        List<Coordinates> coordinatesList,
+        Coordinates target) {
+
+        List<Double> distanceList = coordinatesList.stream()
+                                                   .map(coordinates -> coordinates.calculateDistance(target))
```
Comment (eric-martin) [inline core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsTriple.java]: Same here

File: core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsTriple.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import org.apache.commons.lang3.tuple.ImmutableTriple;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+class MultipleReturnValuesUsingApacheCommonsTriple {
+
+    static ImmutableTriple<Double, Double, Double> getMinAvgMaxTriple(
+        List<Coordinates> coordinatesList,
```
Comment (eric-martin) [inline core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsTriple.java]: two space indent here also

File: core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsPair.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.Comparator;
+import java.util.List;
+
+class MultipleReturnValuesUsingApacheCommonsPair {
+
+    static ImmutablePair<Coordinates, Double> getMostDistantPoint(
+        List<Coordinates> coordinatesList,
```
Comment (eric-martin) [inline core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsPair.java]: two space indent

File: core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsPair.java (hunk 1 above)
Comment (sk1418) [reply core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsPair.java]: fixed all continuation indents, such as long method signatures.
It seems that my IDE (intellij) dosn't think the fluent function calls starting with dot as continuations. Should I make them have indent 2 anyway? (Personally I think the current format is easier to read)

Comment (eric-martin) [reply core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsPair.java]: Yes ... two space indentation for line continuations. If we don't fix it now, we will have to fix it later :)

Comment (sk1418) [reply core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingApacheCommonsPair.java]: Ok. thanks.
I fixed the indentation.

**Pull request merged**